db/python/tables/project.py

```diff
@@ -465,209 +266,71 @@ async def delete_project_data(
 DELETE FROM participant WHERE project = :project;
 DELETE FROM analysis WHERE project = :project;
             """
-            values: dict = {'project': project_id}
-            if delete_project:
-                group_ids = await self.connection.fetch_one(
-                    """
-                    SELECT read_group_id, write_group_id
-                    FROM project WHERE id = :project'
-                    """
-                )
-                _query += 'DELETE FROM project WHERE id = :project;\n'
-                _query += 'DELETE FROM `group` WHERE id IN :group_ids\n'
-                values['group_ids'] = [
-                    group_ids['read_group_id'],
-                    group_ids['write_group_id'],
-                ]
 
             await self.connection.execute(_query, {'project': project_id})
 
-        if delete_project:
-            # pylint: disable=no-member
-            self._get_project_rows_internal.cache_invalidate()
-
         return True
 
-    async def set_group_members(self, group_name: str, members: list[str], author: str):
+    async def set_project_members(
+        self, project: Project, members: list[ProjectMemberUpdate]
+    ):
         """
         Set group members for a group (by name)
         """
 
-        has_permission = await self.gtable.check_if_member_in_group_name(
-            GROUP_NAME_MEMBERS_ADMIN, author
-        )
-        if not has_permission:
-            raise Forbidden(
-                f'User {author} does not have permission to add members to group {group_name}'
-            )
-        group_id = await self.gtable.get_group_name_from_id(group_name)
-        await self.gtable.set_group_members(
-            group_id, members, audit_log_id=await self.audit_log_id()
-        )
-
-    # endregion CREATE / UPDATE
-
-    async def get_seqr_projects(self) -> list[dict[str, Any]]:
-        """
-        Get all projects with meta.is_seqr = true
-        """
-
-        all_projects = await self._get_project_rows_internal()
-        seqr_projects = [p for p in all_projects if p.meta.get('is_seqr')]
-        return seqr_projects
-
-    # gruo
-
-
-class GroupTable:
```

Comment:
I've removed all these methods, as they weren't exposed via any routes and groups are very minimal now, there should only ever be the two admin groups. I'm thinking that we may rethink or remove those admin groups entirely in the future too. But I'm happy to add these back in we reckon they'll be useful in the future.